tests/integration-tests/processors/processors-test.js

```diff
@@ -67,9 +68,27 @@ describe('Processors api', function() {
                 // TODO: when DELETE /processors is implemented, use processorsInserted to empty the table.
             });
         });
-        it('Create processor with type file_download', async () => {
-            nock('https://authentication.predator.dev').get('/?dl=1').reply(200,
-                `{
+        describe('GET /v1/processors/{processor_id}', function () {
+            let processorData, processor;
+            before(async function() {
+                processorData = generateRawJSProcessor('mickeys-processor');
```

Comment:
in the GET /processors PR, you have commented that you would like to change the function's behavior.

I see that you've used it without changing it, does it satisfy the needs after all?

Reply:
i meant to refactor the `before` function where we create many processors to an external function so that we don't have duplicated code in `before` sections. But right now it's not necessary so we can leave it like this